master/internal/rm/multirm/multirm.go

```diff
@@ -0,0 +1,403 @@
+package multirm
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/exp/maps"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/determined-ai/determined/master/internal/rm"
+	"github.com/determined-ai/determined/master/internal/rm/rmerrors"
+	"github.com/determined-ai/determined/master/internal/sproto"
+	"github.com/determined-ai/determined/master/pkg/command"
+	"github.com/determined-ai/determined/master/pkg/model"
+	"github.com/determined-ai/determined/proto/pkg/apiv1"
+	"github.com/determined-ai/determined/proto/pkg/jobv1"
+)
+
+// ErrRPNotDefined returns a detailed error if a resource pool isn't found.
+func ErrRPNotDefined(rp string) error {
+	return fmt.Errorf("could not find resource pool %s", rp)
+}
+
```

Comment:
i'm trusting this is unchanged since it was last landed

Reply:
I had to change the signatures to _not_ take in a RM name, and similarly, in getRM, I removed references to RM name